ipa-core/src/net/server/mod.rs

```diff
@@ -295,18 +280,20 @@ async fn rustls_config(
     for cert in network
         .peers()
         .iter()
-        .filter_map(|peer| peer.certificate.as_ref())
+        .filter_map(|peer| peer.certificate.clone())
     {
         // Note that this uses `webpki::TrustAnchor::try_from_cert_der`, which *does not* validate
         // the certificate. That is not required for security, but might be desirable to flag
         // configuration errors.
         trusted_certs.add(cert)?;
```

Comment:
this is not related to our code, but this change of API makes it less efficient because the certificate gets copied twice - once in our code and the second time inside the `add` method